src/server/system_services/bucket_server.js

```diff
@@ -1400,7 +1410,8 @@ function get_bucket_info({
         namespace: bucket.namespace ? {
             write_resource: pool_server.get_namespace_resource_info(
                 bucket.namespace.write_resource).name,
-            read_resources: _.map(bucket.namespace.read_resources, rs => pool_server.get_namespace_resource_info(rs).name)
+            read_resources: _.map(bucket.namespace.read_resources, rs => pool_server.get_namespace_resource_info(rs).name),
+            caching: bucket.namespace.caching ? bucket.namespace.caching : { ttl: 0 }
```

Comment:
Return undefined if undefined:
```suggestion
            caching: bucket.namespace.caching,
```

Reply:
ok